src/store/mutations.ts

```diff
@@ -1,7 +1,50 @@
 import { RootState } from './RootState';
+import QueryBuilderSparqlGenerator from '@/sparql/QueryBuilderSparqlGenerator';
+import SparqlGenerator from '@/sparql/SparqlGenerator';
 
 export default {
 	setValue( state: RootState, value: string ): void {
 		state.value = value;
+	},
+	runQuery( state: RootState ): void {
+		const prefixes = {
+			wdt: 'http://www.wikidata.org/prop/direct/'
```

Comment:
you can load all prefixes here by loading the allNamespaces variable. OTOH, I'm planning to move that file, so it's fine for now.

Reply:
Done. Please check 4d12ac1b5eb1b3bff8e1eeb1276fd94aee697499